.github/workflows/check_execute

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import subprocess, sys, argparse, time
+
+parser = argparse.ArgumentParser(description="Check the startup of the execute container")
+parser.add_argument("image", help="name of the image you're running")
+parser.add_argument("--client_ipcpath", help="IPC Path of the geth container", required=True)
+parser.add_argument("--database_hostname", help="hostname of the postgres instance", default="postgres")
+parser.add_argument("--database_name", help="name of the postgres database", default="postgres")
+parser.add_argument("--database_port", help="port of the postgres database", default="5432")
+parser.add_argument("--database_user", help="username of the postgres database user", default="postgres")
+parser.add_argument("--database_password", help="password of the postgres database user", default="postgres")
+parser.add_argument("--wait_time", help="seconds to wait for the container to be ready", default="60", type=int)
```

Comment:
This strikes me as a potentially low number - any ideas on how long it's taking for the container to be ready right now?

Reply:
About 30 seconds. Surprised me too - you can see it in the logs on the actions tab.